doc/python_api_reference_vDev.md

```diff
@@ -7714,20 +7714,18 @@ def cnot(
 def copy(
     self,
     *,
-    fresh_entropy: bool = False,
+    fresh_entropy: bool = True,
     seed: Optional[int] = None,
 ) -> stim.TableauSimulator
 
     """Returns a copy of the simulator. A simulator with the same internal state.
 
     Args:
-        fresh_entropy: If False and no seed is specified, the RNG state is copied
-            together with the rest of the simulator. The two copies will produce
-            the same measurement outcomes for the same quantum circuits. This is
-            the default behavior. If False and seed is specified, the RNG is
-            reinitialized using the new seed. If True and no seed is specified,
-            the RNG is reinitialized using a random seed. If True and seed is
-            specified, exception is raised. Defaults to False.
+        fresh_entropy: If False, the RNG state is copied together with the rest
```

Comment:
Rephrase so that the default option sounds passive instead of active, and the modified case sounds like an active intervention instead of a passive standing by. The user specifying this argument isn't asking for fresh entropy, that's what happens by default. The user specifying this argument wants to block fresh entropy being used, they want to keep the internal rng state. E.g. I'd call this perhaps `also_copy_prng_state`.

Reply:
Done.

I phrased it in terms of how the new RNG is initialized in each case. I don't think we can make it any more passive than that since the generation of a new random seed can't be thought of as "copying" anything.